samples/unity-of-bugs/Assets/Scripts/BugFarm.cs

```diff
@@ -6,121 +6,124 @@
 using UnityEngine;
 using UnityEngine.Assertions;
 
-public class BugFarm : MonoBehaviour
+namespace Assets.Scripts
```

Comment:
Do we really need a namespace?

Because this is a sample for customers to learn about Sentry, I wanted to keep this as simple as possible and originally didn't add a namespace.

Reply:
I thought you just missed the namespace in the first place :)

If you don't want it, we can remove it. But personally, I'd prefer to leave it as is and not in `global::` scope. 

Remove?

Reply:
Sorry I got back from PTO today. I think it's worth keeping these samples (that go into the user games through the UPM sample feature) simple and remove the namespace altogether

Reply:
ok, will remove namespaces then